ee/clickhouse/models/test/__snapshots__/test_cohort.ambr

```diff
@@ -0,0 +1,455 @@
+               FROM
+                 (SELECT distinct_id,
+                         person_id,
+                         max(_timestamp) as _timestamp
+                  FROM person_distinct_id
+                  WHERE team_id = 2
+                  GROUP BY person_id,
+                           distinct_id,
+                           team_id
+                  HAVING max(is_deleted) = 0)
+               GROUP BY distinct_id) as pdi ON events.distinct_id = pdi.distinct_id
+            WHERE team_id = 2
+              AND timestamp >= '2020-01-07 00:00:00'
+              AND timestamp <= '2020-01-10 00:00:00'
+              AND ((event = '$pageview'))
+            GROUP BY person_id
+            HAVING count(*) = 1) )
+  '
+---
+# name: TestCohort.test_cohortpeople_action_count.11
+  '
+  SELECT person_id
+  FROM cohortpeople
+  where cohort_id = 2
+  '
+---
+# name: TestCohort.test_cohortpeople_action_count.12
+  '
+  
+  SELECT count(*)
+  FROM
+    (SELECT 1
+     FROM cohortpeople
+     WHERE team_id = 2
+       AND cohort_id = 2
+     GROUP BY person_id,
+              cohort_id,
+              team_id
+     HAVING sum(sign) > 0)
+  '
+---
+# name: TestCohort.test_cohortpeople_action_count.13
+  '
+  
+  SELECT count(*)
+  FROM
+    (SELECT 1
+     FROM cohortpeople
+     WHERE team_id = 2
+       AND cohort_id = 2
+     GROUP BY person_id,
+              cohort_id,
+              team_id
+     HAVING sum(sign) > 0)
+  '
+---
+# name: TestCohort.test_cohortpeople_action_count.14
+  '
+  
+  SELECT DISTINCT p.id
+  FROM
+    (SELECT *
+     FROM person
+     JOIN
+       (SELECT id,
+               max(_timestamp) as _timestamp,
+               max(is_deleted) as is_deleted
+        FROM person
+        WHERE team_id = 2
+        GROUP BY id) as person_max ON person.id = person_max.id
+     AND person._timestamp = person_max._timestamp
+     WHERE team_id = 2
+       AND person_max.is_deleted = 0 ) AS p
+  INNER JOIN
+    (SELECT distinct_id,
+            argMax(person_id, _timestamp) as person_id
+     FROM
+       (SELECT distinct_id,
+               person_id,
+               max(_timestamp) as _timestamp
+        FROM person_distinct_id
+        WHERE team_id = 2
+        GROUP BY person_id,
+                 distinct_id,
+                 team_id
+        HAVING max(is_deleted) = 0)
+     GROUP BY distinct_id) AS pdi ON p.id = pdi.person_id
+  WHERE team_id = 2
+    AND pdi.distinct_id IN
+      (SELECT distinct_id
+       FROM
+         (SELECT distinct_id,
+                 argMax(person_id, _timestamp) as person_id
+          FROM
+            (SELECT distinct_id,
+                    person_id,
+                    max(_timestamp) as _timestamp
+             FROM person_distinct_id
+             WHERE team_id = 2
+             GROUP BY person_id,
+                      distinct_id,
+                      team_id
+             HAVING max(is_deleted) = 0)
+          GROUP BY distinct_id)
+       WHERE person_id IN
```

Comment:
This seems pretty suspect to me.

I think we can simplify the negation query to simply be `person_id NOT IN ({extract_person})` ?